src/schema/schema.graphql

```diff
@@ -280,10 +280,36 @@ type AuthResponse {
 	skin: [Skin]
 }
 
+type ScratchCode{
+	ScratchCode: String
+}
+
+type EnableTwoFactorAuthResponse {
+	secret: [Secret]
+	scratchCodes: [ScratchCode]
+	authToken: [TwoFactorAuthToken]
+}
+
 type ProfileImageChangeResponse {
 	itemId: ID
 }
 
+type TwoFactorAuthToken {
+	_content: String,
+	verifyAccount: Boolean,
```

Comment:
not sure if we are going to use these two properties

Reply:
done